docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -206,8 +206,52 @@ throughout this style guide.
    Include visual of 'Do's and Don'ts!
 
 
-2.0 Dealing with Limitations in Upstream Research
-+++++++++++++++++++++++++++++++++++++++++++++++++
+2.0 Research question
++++++++++++++++++++++
+
+As much as possible, our research should be precise about the variables of interest.
+For example, if we are modeling pregnancy, the variable we are interested in is the
+ability to become pregnant.
+This not only clarifies that we are interested in biological sex and not gender, but is even
+more precise that it is this facet of biological sex that matters,
+not another facet such as chromosomal makeup.
+
+It is frequently not possible to be this precise;
+we should try to at least determine whether our research question is about sex or gender.
+In some cases, even this may not be clear, for example if we are researching an
+association that could be mediated by either sex or gender (or both), making them both
+variables of interest.
+
+2.1 Proxy measures
```

Comment:
I think this might fit better in the "data collection" (or even "data analysis"?) section, but I'm not sure.